typescript/sdk/src/consts/multisigIsm.ts

```diff
@@ -888,6 +894,10 @@ export const defaultMultisigConfigs: ChainMap<MultisigConfig> = {
         address: '0xf0da628f3fb71652d48260bad4691054045832ce',
         alias: 'Luganodes',
       },
+      {
```

Comment:
noting the recommended threshold here would've been 5
https://discord.com/channels/935678348330434570/1276645440149786644/1284149950174724190

Reply:
discussed offline, will be updating validator tooling to enforce the new thresholds from next batch onwards